namui/namui-cli/src/services/resource_collect_service.rs

```diff
@@ -149,33 +149,46 @@ impl CollectOperation {
         release_path: impl AsRef<std::path::Path>,
     ) -> Result<()> {
         let src_path = project_root_path.as_ref().join(&self.src_path);
-        let dest_path = release_path.as_ref().join(&self.dest_path);
-        copy_resource(src_path, dest_path)
+        let dest_dir_path = release_path.as_ref().join(&self.dest_dir_path);
+        copy_resource(src_path, dest_dir_path)
+    }
+
+    pub fn dest_path(&self) -> PathBuf {
+        self.dest_dir_path.join(self.src_path.file_name().unwrap())
     }
 }
 
-fn copy_resource(from: impl AsRef<std::path::Path>, to: impl AsRef<std::path::Path>) -> Result<()> {
+fn copy_resource(
+    from: impl AsRef<std::path::Path>,
+    dest_dir_path: impl AsRef<std::path::Path>,
+) -> Result<()> {
     let from = from.as_ref();
-    let to = to.as_ref();
-    debug_println!("resource_collect_service: copy {:?} -> {:?}", &from, &to);
-    ensure_dir(to)?;
-    match from.is_dir() {
-        true => copy_dir(from, to),
-        false => copy_file(from, to),
-    }
+    let dest_dir_path = dest_dir_path.as_ref();
+    debug_println!(
+        "resource_collect_service: copy {:?} -> {:?}/{:?}",
+        &from,
+        &dest_dir_path,
+        &from.file_name().unwrap()
+    );
+    ensure_dir(dest_dir_path)?;
+    assert!(dest_dir_path.is_file());
```

Comment:
You mean `assert!(dest_dir_path.is_dir());`?

Reply:
oh, `assert!(from.is_file())`;

Reply:
```suggestion
    assert!(from.is_file());
```